connector_jira_tempo/data/cron.xml

```diff
@@ -4,11 +4,11 @@
 -->
 <odoo noupdate="1">
 
-  <record forcecreate="True" id="ir_cron_jira_sync_jira_tempo_status" model="ir.cron">
-    <field name="name">JIRA - Sync Timesheet approval status</field>
+  <record forcecreate="True" id="ir_cron_jira_sync_tempo_timesheets_approval_status" model="ir.cron">
```

Comment:
you should add a migration step for this and any other change that alters existing records

Reply:
Is there a good example of similar case?

Reply:
https://odoo-development.readthedocs.io/en/latest/maintenance/data-migration.html

get an env, find the old ir.model.data record and change its name